src/common/transformations/tests/common_optimizations/group_normalization_fusion_tests.cpp

```diff
@@ -0,0 +1,570 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/data_utils.hpp"
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/group_normalization.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/mvn.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/serialize.hpp"
+#include "transformations/common_optimizations/group_normalization_fusion.hpp"
+#include "transformations/init_node_info.hpp"
+
+using namespace testing;
+using namespace ov;
+
+template <element::Type_t T_act_elem,
+          element::Type_t T_gn_gamma_elem = T_act_elem,
+          element::Type_t T_gn_beta_elem = T_act_elem,
+          element::Type_t T_in_gamma_elem = T_act_elem,
+          element::Type_t T_in_beta_elem = T_act_elem>
+class GroupNormalizationFusionTestsFixture
+    : public ::testing::TestWithParam<
```

Comment:
```suggestion
    : public TestWithParam<
```
As there is global `using namespace testing;` or remove this global using and pull required classes from gtest

Reply:
Done